crates/controller/src/mouse.rs

```diff
@@ -212,16 +212,47 @@ enum DragResolution {
     Rect(ScreenRect),
 }
 
-fn update_position(windows: Res<Windows>, mut mouse: ResMut<MousePosition>) {
+fn update_position(
+    windows: Res<Windows>,
+    hud: Query<(&GlobalTransform, &Node), With<HudVisibleNode>>,
+    mut mouse: ResMut<MousePosition>,
+) {
     let window = windows.get_primary().unwrap();
     mouse.set_position(
         window
             .cursor_position()
+            .and_then(|position| {
+                let win_size = Vec2::new(window.width(), window.height());
+                let found = hud.iter().find(|(transform, node)| {
+                    is_in(transform, node, win_size, position)
+                });
+                if found.is_none() { Some(position) } else { None }
+            })
             .map(|position| position / Vec2::new(window.width(), window.height()))
             .map(|normalised_position| normalised_position.clamp(Vec2::ZERO, Vec2::ONE)),
     );
 }
 
+fn is_in(
+    box_transform: &GlobalTransform,
+    box_size: &Node,
+    win_size: Vec2,
+    mouse_position: Vec2,
+) -> bool {
+    // WARNING: This is because mouse y starts on bottom, GlobalTransform on top
+    let mouse_position = Vec2::new(mouse_position.x, win_size.y - mouse_position.y);
+
+    let box_size = box_size.size();
+    let box_transform: Vec3 = box_transform.translation();
+    // WARNING: This is because GlobalTransform is centered, width/2 to left and to right, same on vertical
+    let box_position = Vec2::new(box_transform.x, box_transform.y) - box_size / Vec2::splat(2.);
```

Comment:
* `Vec2::new(box_transform.x, box_transform.y)` -> `box_transform.xy()` should work.
* Isn't `box_size / Vec2::splat(2.)` the same as `box_size / 2.`?